test/test_other.py

```diff
@@ -406,6 +402,13 @@ def test_emcc_basics(self, compiler, suffix):
     self.assertContained(["error: use of undeclared identifier 'cheez", "error: unknown type name 'cheez'"], stderr)
     self.assertContained('errors generated.', stderr.splitlines()[-2])
 
+  def test_dumpmachine(self):
```

Comment:
Should this use parameterised helper maybe instead of duplicating?

Reply:
I feel like this type/level of duplication in tests is pretty reasonable. 

I'm a big fan of parameterized tests, but somehow in this case it seems live overkill and would actually harm readability.

Reply:
Okay not a big difference either way.